cmd/serf/command/agent/command.go

```diff
@@ -452,6 +463,7 @@ func (c *Command) startAgent(config *Config, agent *Agent,
 	c.Ui.Info(fmt.Sprintf("     Encrypted: %#v", agent.serf.EncryptionEnabled()))
 	c.Ui.Info(fmt.Sprintf("      Snapshot: %v", config.SnapshotPath != ""))
 	c.Ui.Info(fmt.Sprintf("       Profile: %s", config.Profile))
+	c.Ui.Info(fmt.Sprintf("       Message Compression Enabled: %v", config.EnableCompression))
```

Comment:
Another nit: this breaks the right-aligned formatting which is useful for quick skimming as well as just looking odd. I'd suggest we shoot for the following output instead:

```
           Encrypted: true
            Snapshot: /foo/bar
             Profile: ...
 Messages Compressed: true
```

That is with an extra six spaces prefix the original items and a single space prefix for the new one to differentiate for other log lines below.